app/build.gradle

```diff
@@ -95,6 +96,11 @@ dependencies {
     kapt "com.google.dagger:hilt-compiler:$hilt_version"
     implementation "androidx.hilt:hilt-navigation-compose:1.1.0"
 
+    //room dependencies
+    kapt "androidx.room:room-compiler:$room_version"
```

Comment:
I recommend to use KSP (Kotlin Symbol Processing), which is a faster alternative to kapt. Both Dagger and Room support that.

I explain it here: https://www.youtube.com/watch?v=WVhgUGyUnhQ